python/ray/serve/tests/test_failure.py

```diff
@@ -209,6 +210,41 @@ def __call__(self):
                 time.sleep(0.1)
 
 
+def test_create_backend_idempotent(serve_instance):
+    serve.init()
+
+    def f():
+        pass
+
+    controller = serve.api._get_controller()
+
+    replica_config = ReplicaConfig(f)
+    backend_config = BackendConfig({"num_replicas": 1})
+
+    for i in range(10):
+        controller.create_backend.remote("my_backend", backend_config,
+                                         replica_config)
+
+    assert len(ray.get(controller.get_all_backends.remote())) == 1
```

Comment:
Should we be doing any other correctness checks here? e.g., making an HTTP request and verifying that it returns the expected output?

Reply:
Sounds good, I'll add that

Reply:
Oh, right, I was reading this as if it were the public API.